content/en/docs/releases/kubeflow-1.6.md

```diff
@@ -22,7 +22,7 @@ weight = 98
           <a href="https://blog.kubeflow.org/kubeflow-1.6-release/">Kubeflow 1.6 Release Announcement</a>
         <br>
         <b>Video:</b> 
-          TBA
+          <a href="https://youtu.be/RR1xSfnFGGI">Kubeflow 1.6 Release Overview</a>
```

Comment:
Nit: can you use the expanded link: https://www.youtube.com/watch?v=RR1xSfnFGGI

Reply:
Fixed